app/scripts/modules/amazon/src/serverGroup/details/scalingPolicy/upsert/alarm/metricSelector.component.ts

```diff
@@ -95,6 +95,12 @@ export class MetricSelectorController implements IController {
         }
         if (selected) {
           this.state.selectedMetric = selected;
+        } else {
+          // If metricName is blank (new policy), try to find a CPU metric or select the first option instead of sitting on the invalid blank option
+          if (alarm.metricName === '' && this.state.metrics.length > 0) {
```

Comment:
```suggestion
          if (!alarm.metricName && this.state.metrics.length) {
```
just think of the savings